trusttrees.py

```diff
@@ -138,15 +141,15 @@ def wrapper_of_registar_function(input_domain):
                 break
             time.sleep(1)
 
-        return status == 'available'
+        return status.startswith('available')
```

Comment:
If there is some unexpected source of false-positives with this, we can use an enum, and add a color for `available_reserved` or `available_preorder`, but right now I'm okay with kicking the can for that. I can make another GitHub issues later.